hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/net/TestNetUtils.java

```diff
@@ -111,6 +111,7 @@ public void testInvalidAddress() throws Throwable {
       fail("Should not have connected");
     } catch (UnknownHostException uhe) {
       LOG.info("Got exception: ", uhe);
+      assertEquals("invalid-test-host:0", uhe.getMessage());
```

Comment:
Use `GenericTestUtils.assertExceptionContains()`; this rethrows the exception if there's no match, and allows for extra text in the messge